test/sanity/ignore.txt

```diff
@@ -315,6 +315,9 @@ test/integration/targets/pip/files/ansible_test_pip_chdir/__init__.py future-imp
 test/integration/targets/pip/files/ansible_test_pip_chdir/__init__.py metaclass-boilerplate
 test/integration/targets/pip/files/setup.py future-import-boilerplate
 test/integration/targets/pip/files/setup.py metaclass-boilerplate
+test/integration/targets/plugin_loader/ignore_exts/library/demo.py future-import-boilerplate
```

Comment:
I think we're usually adding the boilerplate on new tests rather than adding new ignores

Reply:
Going to just do this differently and make it part of `module_precedence` tests instead.